app/src/main/java/com/example/teamyuml/decentworkmobile/Login.java

```diff
@@ -1,4 +1,136 @@
 package com.example.teamyuml.decentworkmobile;
 
-class Login {
+import android.app.ProgressDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Window;
+import android.widget.Button;
+import android.widget.EditText;
+import android.content.Intent;
+import android.view.View;
+import android.widget.Toast;
+import com.android.volley.Request;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+ * Class creating connection with server
+ * Checking adres email
+ * Parsing data to JSON and sending to the server
+ */
+
+public class Login extends AppCompatActivity {
+
+
+    ProgressDialog progressDialog;
+    private EditText loginInputEmail, loginInputPassword;
+    private Button btnlogin;
+    private Button btnLinkSignup;
+    private static final String TAG = "Login";
+    private static final String URL_FOR_LOGIN = "Heve we wanna url adress for Login";
```

Comment:
I think the best option for us is to keep something like private static final BASE_URL = 'http://localhost:8000/' + getBASE_URL() public method and then in the Login class URL_FOR_LOGIN = VolleyInstance.getBASE_URL() + '/someendpoint/'.

Reply:
Amm I tried IT Earlier and that doesn't working :(